.metadata/.log

```diff
@@ -0,0 +1,161 @@
+!SESSION 2021-09-02 10:43:34.168 -----------------------------------------------
```

Comment:
@leonardo329cf,
This file shouldn't be commited.
Remove all files from folder `.metadata/`

Reply:
@luizfmarins 
Done,
.metadata folder removed and added to .gitignore.
Thanks, you may continue the review.